client/src/StoresPage.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Store = ({ addToCart }) => {
+    const navigate = useNavigate();
+
+    // Mock Data for Categories and Products
+    const categories = {
+        Fruits: [
```

Comment:
We can implement these using more categories! maybe when the server side is completely done!!